src/v/kafka/server/handlers/handler_probe.cc

```diff
@@ -102,6 +103,14 @@ void handler_probe::setup_metrics(
           sm::description("Number of bytes sent in kafka replies"),
           labels)
           .aggregate(aggregate_labels),
+        sm::make_histogram(
```

Comment:
We def want this in internal metrics too but it can wait for a patch release I guess.

Reply:
It is in internal metrics. Used the public metrics histogram by accident though. Will switch it to the internal metrics histogram as it has more resolution on the lower bound.